Resources/Prototypes/Entities/Clothing/Uniforms/base_clothinguniforms.yml

```diff
@@ -50,6 +50,11 @@
   - type: WirelessNetworkConnection
     range: 1200
   - type: StationLimitedNetwork
+  - type: Tag #Delta-V : Adds tags to fix the Syndicate Jumpskirt, which is parented to this? this needs to be updated with *every* new tag given to UnsensoredClothingUniformBase, as tags aren't inherited
```

Comment:
not sure if i'm misreading but the issue is that a child receiving tags overwrites the tags of the parent, this usually works however as Harpies need a "skirt" tag to wear their associated skirts, we're overwriting the parent tag to add the skirt tag in. In an ideal world, we should be able to just inherit the parent tag and add the skirt tag over the top. 

Syndicate Jumpskirt parents off of this sensor-off version, to my understanding so the person making it can make a funny "haha they're evil so their sensor is off" code comment. Other solution aside from this gross tag nonsense is to just reparent the Syndicate Jumpskirts to ClothingUniformSkirtBase below this.

Reply:
Tag code is pretty disgusting throughout the repo, inheritance would be nice but would instead cause problems when you *do* want to overwrite tags.

Reply:
Added it to the wrong entity :O, i'm adding the skirt tag to: `UnsensoredClothingUniformSkirtBase` , in order to let Harpies wear the Syndicate Jumpskirt, I have to add the other tags in order to not overwrite the other tags needed.

To ClothingUniformSkirtBase, I'm adding the WhitelistChameleon tag, so the Chameleon jumpsuit can now disguise as skirts again.

To the Chameleon jumpsuit, I'm adding the skirt tag so Harpies can put it on first without having to disguise it as something else.